enos/modules/target_ec2_instances/variables.tf

```diff
@@ -24,6 +24,12 @@ variable "common_tags" {
   default     = { "Project" : "vault-ci" }
 }
 
+variable "disable_selinux" {
```

Comment:
Since we modified this interface we should also update the other target modules so that if for some reason we decide to try spot instances again it should work.